proto/common/v1/info.proto

```diff
@@ -1,75 +1,15 @@
 syntax = "proto3";
 
 // Package common defines commonly used types agnostic to the node role on the Sonr network.
-package sonrio.motor.common.v1;
+package sonrio.common.v1;
 
-// Direction is the direction of a stream.
-enum Direction {
-  // Unspecified is the default value.
-  DIRECTION_UNSPECIFIED = 0;
-
-  // Incoming is the direction of an incoming stream.
-  DIRECTION_INCOMING = 1;
-
-  // Outgoing is the direction of an outgoing stream.
-  DIRECTION_OUTGOING = 2;
-}
-
-// Standard MIME with Additional Extensions
-message MIME {
-  // Type of File
-  Type type = 1;
-
-  // Extension of File
-  string subtype = 2;
-
-  // Type/Subtype i.e. (image/jpeg)
-  string value = 3;
-
-  // File Content Type
-  enum Type {
-    // Other File Type - If cannot derive from Subtype
-    TYPE_UNSPECIFIED = 0;
-
-    // Sound, Audio Files
-    TYPE_AUDIO = 1;
-
-    // Document Files - PDF, Word, Excel, etc.
-    TYPE_DOCUMENT = 2;
-
-    // Image Files
-    TYPE_IMAGE = 3;
-
-    // Text Based Files
-    TYPE_TEXT = 4;
-
-    // Video Files
-    TYPE_VIDEO = 5;
-
-    // URL Links
-    TYPE_URL = 6;
-
-    // Crypto Files
-    TYPE_CRYPTO = 7;
-  }
-}
+option go_package = "github.com/sonr-io/sonr/thirdparty/types/common";
```

Comment:
Can you add the java package options to these protos to match the other protos?

Reply:
this is sick idea

Reply:
we can do it for all motor targets

Reply:
Do you have current Java package options?

Reply:
Look for example in `proto/motor/v1/request.proto`

Reply:
Defered for #522